core/src/main/scala/lucuma/sbtplugin/LucumaPlugin.scala

```diff
@@ -127,7 +123,6 @@ object LucumaPlugin extends AutoPlugin {
     )
 
     lazy val lucumaGitSettings = Seq(
-      // TODO replace with `useConsoleForROGit := true`
       git.gitUncommittedChanges := {
```

Comment:
I was hoping we could switch this but it creates some other annoying behaviors ...